ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -239,6 +240,31 @@ const moduleLabels: Map<AccessModule, JSX.Element> = new Map([
   [AccessModule.DATAUSERCODEOFCONDUCT, <div>Sign Data User Code of Conduct</div>],
 ]);
 
+const withAborter = async(abortableFn: (AbortController) => void) => {
+  const aborter = new AbortController();
+  try {
+    await abortableFn(aborter);
+  } catch (e) {
+    if (!isAbortError(e)) { throw e; }
```

Comment:
opt nit: would generally prefer newlines for if statements, but it's not a hard/fast rule in the linter